pom.xml

```diff
@@ -202,15 +211,6 @@
                             <goal>jjtree-javacc</goal>
                         </goals>
                     </execution>
-                    
-                    <!-- execute JJTree explicitely in order to generate the *.jj file needed for JJDoc -->
```

Comment:
Will removing this cause maven not to generate the source anymore when I just changes simple grammar rules?